data/palmabit.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "Palmabit",
+  "url": "https://www.palmabit.com",
+  "career_page_url": "https://www.palmabit.com/it/work-with-us",
+  "type": "Consulting",
+  "categories": ["cloud_software"],
+  "remote_policy": "Optional",
```

Comment:
@edoardocostantinidev is not enough this policy?

Reply:
Optional means that the company allows you to choose when to work remotely or in office, and can also ask you to go sometimes. I couldn't find any info that suggested such format. If this is your current situation it's ok too and we can approve.